core/endorser/endorser_test.go

```diff
@@ -1174,38 +1224,40 @@ var _ = Describe("Endorser", func() {
 			Expect(proto.Equal(
 				proposalResponse.Interest,
 				&pb.ChaincodeInterest{
-					Chaincodes: []*pb.ChaincodeCall{{
-						Name:            "myCC",
-						CollectionNames: []string{"mycollection-1"},
-						NoPrivateReads:  true,
-						KeyPolicies:     []*cb.SignaturePolicyEnvelope{sbe1, sbe2},
-					}},
+					Chaincodes: []*pb.ChaincodeCall{
+						{
+							Name:                     "myCC",
+							KeyPolicies:              []*cb.SignaturePolicyEnvelope{sbe},
+							DisregardNamespacePolicy: true,
+						},
+						{
+							Name:            "myCC",
+							CollectionNames: []string{"mycollection-1"},
+							NoPrivateReads:  true,
+						},
+					},
 				},
 			)).To(BeTrue())
 		})
 
-		It("add SBE only", func() {
-			sbe1 := &cb.SignaturePolicyEnvelope{
+		It("SBE only, no chaincode policy updates", func() {
```

Comment:
I hate to nit-pick of this level but I noticed at a couple of places and just for your reference -  When following ginkgo pattern, an `It` is expected to sound like it makes sense as an sentence, in conjunction with enclosing `Describe` and `Context`. In other word, `It` is expected to be part of the sentence :-)

For example, in this case, following may make sense
```
Describe("Endorser",
Context("when building the ChaincodeInterest"
It("disregards chaincode policy if all updates are driven by SBE", func() {
```